app/players/player.py

```diff
@@ -0,0 +1,14 @@
+from abc import abstractmethod, ABC
+
+
+class Player(ABC):
+    def __init__(self, nickname: str) -> None:
+        self.nickname = nickname
+
+    @abstractmethod
+    def get_rating(self) -> None:
+        pass
+
+    @abstractmethod
+    def player_info(self) -> None:
```

Comment:
Are you sure that these two functions return everywhere None? these methods will return specific types